examples/extensions/commands/cmd_clean.py

```diff
@@ -0,0 +1,44 @@
+from conan.api.conan_api import ConanAPIV2
+from conans.cli.command import conan_command, OnceArgument
+from conans.cli.output import Color, ConanOutput
+
+
+info_color = Color.BRIGHT_WHITE
+removed_color = Color.BRIGHT_YELLOW
+
+
+@conan_command(group="Custom commands")
+def clean(conan_api: ConanAPIV2, parser, *args):
+    """
+    Clean the earlier recipe revisions (and their packages) and the packages ones from local or remote.
+
+    Important! Keeping the LATEST package revisions belonging to the LATEST recipe revision.
+    """
+    parser.add_argument('-r', '--remote', action=OnceArgument,
+                        help='Will remove from the specified remote')
+    args = parser.parse_args(*args)
+    remote = conan_api.remotes.get(args.remote) if args.remote else None
+    output_remote = remote or "Local cache"
+    # Ouput object
+    out = ConanOutput()
+    # Getting all the recipe revisions and the latest ones
+    all_rrevs = conan_api.search.recipe_revisions("*/*#*", remote=remote)
+    latest_rrevs = conan_api.search.recipe_revisions("*/*#latest", remote=remote)
+    for rrev in all_rrevs:
```

Comment:
This is not necessary, as ``conan_api.remove.recipe(rrev, remote=remote)`` is doing the right thing, removing packages too

Reply:
I don't get what you mean. I'm removing the latest package revisions from the latest recipe revisions, and if you see below, I have these lines:

```python
else:
            conan_api.remove.recipe(rrev, remote=remote)
            out.writeln(f"Removed RREV: {rrev.repr_notime()} [{output_remote}]", fg=removed_color)
```